R/removeat.R

```diff
@@ -124,21 +134,11 @@ indicesToRetain <- function(names, at, length.x, ignore.case = TRUE, split = NUL
         return(1:length.x)
     # 'at' is character and able to represent a variable
     if (is.character(at))
-    {
-        at <- TrimWhitespace(at)
-        if (ignore.case)
-        {
-            at <- tolower(at)
-            names <- tolower(names)
-        }
-        if (!is.null(split))
-            at <- ConvertCommaSeparatedStringToVector(at, split)
-        return(!names %in% at)
-    }
+        return(determineIndicesFromChar(at, names, ignore.case, split))
     # 'at' is numeric
     if (anyNA(at) || !AllIntegers(at))
         stop("'at' must contain character (string) or integer values.")
-    if (any(is.na(at) | at < 1))
+    if (any(at < 0))
```

Comment:
Why change from `at < 1` to `at < 0`?

Reply:
Typo, Fixed in 403ae35584b10b692a343945c3b34c18c89c38d1  thanks